index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/font-awesome/css/font-awesome.min.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <nav class="flex-nav">
+        <h1 class="logo">My Logo</h1>
+        <div class="toggleNav"><i class="fa fa-bars"></i></div>
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <section>
+        <h1>I&rsquo;m Francis.<br />Glad to see you!</h1>
```

Comment:
Your design
![image](https://github.com/lukademinnosa/module1_project1/assets/78251927/81fdf43a-3e05-44a9-8cf6-679247b36c39)

Figma
![image](https://github.com/lukademinnosa/module1_project1/assets/78251927/b5912aa9-3f67-498b-8043-51b245a49792)

I would suggest that we increase the `font-weight` of the main title "I'm Francis. ... " and use the same text color according to the given design. 👍